charts/spark-operator-chart/templates/spark-rbac.yaml

```diff
@@ -49,7 +49,6 @@ subjects:
 roleRef:
   kind: Role
   name: spark-role
-  namespace: {{ $jobNamespace }}
```

Comment:
@ananth102 This was added as a part of the support of multiple namespace here https://github.com/kubeflow/spark-operator/commit/835281d4d3110d265687a9d5b7504a261538af10. Did you validate if this is working for you? Please see the PR raised by @Aransh

Reply:
it's not needed, thanks for removing it